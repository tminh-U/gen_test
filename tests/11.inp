13
20 13 3 20 9 11 16 9 15 3 3 3 19 
10
5 8
7 9
10 13
6 7
4 5
11 12
6 9
8 8
13 13
4 8
