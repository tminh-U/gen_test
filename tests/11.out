10
5
16
4
3
4
8
2
2
11
